api/backend.proto

```diff
@@ -150,4 +154,17 @@ service BackendService {
       body: "*"
     };
   }
+
+  // ReleaseAllTickets moves all tickets from the pending state, to the active
+  // state. This enables them to be returned by query, and find different
+  // matches.
+  // 
+  // BETA FEATURE WARNING:  This call and the associated Request and Response
+  // messages are not finalized and still subject to possible change or removal.
+  rpc ReleaseAllTickets(ReleaseAllTicketsRequest) returns (ReleaseAllTicketsResponse) {
```

Comment:
Use google.protobuf.Empty instead?

Reply:
Thought about it, but I'd prefer to stick with having dedicated request/response types.
The exception is for CRUD operations, which follows the API guide.